changelog/59079.removed

```diff
@@ -0,0 +1,2 @@
+Remove the glance state and execution module in favor of the glance_image
+state module and the glanceng execution module.
```

Comment:
**question**: Does towncrier behave nicely with newlines?

**suggestion**: "Removed the **deprecated** glance state...." -- noting that the module was deprecated should make things less surprising for readers :upside_down_face: (it might be obvious for module users, but it wasn't for me - I had to go find the deprecation warning in this PR)

Reply:
yes towncrier behaves with newlines:

```
(salt-3.6)  ch3ll@megan-precision5550  ~/git/salt   remove_glance  nox -e 'changelog(draft=True)' -- 3003
nox > Running session changelog(draft=True)
nox > Re-using existing virtual environment at .nox/changelog-draft-true.
nox > python -m pip install --progress-bar=off -r requirements/static/ci/py3.6/changelog.txt
nox > towncrier --version=3003 --draft
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

Salt 3003 (2021-03-17)
======================


Removed
-------

- Remove the glance state and execution module in favor of the glance_image
  state module and the glanceng execution module. (#59079)

nox > Session changelog(draft=True) was successful.
```